Auth0/Authentication.swift

```diff
@@ -95,32 +93,30 @@ public protocol Authentication: Trackable, Loggable {
        }
     ```
 
-    You can also specify audience, scope, and additional parameters
+    You can also specify audience and scope
 
     ```
     Auth0
        .authentication(clientId: clientId, domain: "samples.auth0.com")
        .login(phoneNumber: "+4599134762367",
             code: "123456",
             audience: "https://myapi.com/api",
-            scope: "openid email",
```

Comment:
For a later PR, but make sure you have instructions somewhere for adding custom params to `/authorize`